src/mcp/tools/dataconnect/execute_dataconnect_mutation.ts

```diff
@@ -0,0 +1,64 @@
+import { z } from "zod";
+
+import { tool } from "../../tool.js";
+import { mcpError } from "../../util.js";
+import * as client from "../../../dataconnect/dataplaneClient.js";
+import { pickService } from "../../../dataconnect/fileUtils.js";
+import { graphqlResponseToToolResponse } from "./converter.js";
+
+export const execute_dataconnect_mutation = tool(
+  {
+    name: "execute_dataconnect_mutation",
+    description: "Executes a deployed Data Connect query or mutation. Can read and write data.",
```

Comment:
This is only for mutation, right?

Reply:
Huh, I didn't realize that it rejects queries, I thought it was just that we had a different endpoint for readonly clients. I'll reword this to be clearer